content/en/blog/2022/2022-01-06-meet-our-contributors-APAC-India-region-01.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (India region)"
+date: 2022-01-06
+slug: meet-our-contributors-APAC-India-01
+---
+
+**Authors & Interviewers:** [Anubhav Vardhan](https://github.com/anubha-v-ardhan), [Atharva Shinde](https://github.com/Atharva-Shinde), [Avinesh Tripathi](https://github.com/AvineshTripathi), [Debabrata Panigrahi](https://github.com/Debanitrkl), [Kunal Verma](https://github.com/verma-kunal), [Pranshu Srivastava](https://github.com/PranshuSrivastava), [Pritish Samal](https://github.com/CIPHERTron), [Purneswar Prasad](https://github.com/PurneswarPrasad), [Vedant Kakde](https://github.com/vedant-kakde)
+
+**Editor:** [Priyanka Saggu](psaggu.com)
+
+---
+
+Good day, everyone :wave: 
+
+Welcome to the first episode of the APAC edition of the "Meet Our Contributors" blog post series.
+
+
+In this post, we'll meet five amazing folks from the India region who have been actively contributing to the Upstream Kubernetes projects in a variety of ways, as well as being the leaders or maintainers of numerous community initiatives.
+
+:dizzy: *Let's get started without further ado ...*   
+
+
+## [Arsh Sharma](https://github.com/RinkiyaKeDad)
+
+Arsh was recently employed with VMware as a Member of Technical Staff. As a new contributor, he realised that 1:1 mentorship opportunities were quite beneficial in getting him started with the upstream project.
+
+He is presently a CI Signal shadow on the Kubernetes 1.23 release team. He is also contributing to the sig-testing and sig-docs projects, as well as the [cert-manager](https://github.com/cert-manager/infrastructure) tools development work that is being done under the sig-architecture.
+
+To the newcomers, Arsh helps plan their early contributions sustainably.
+
+> "I would encourage folks to contribute in a way that's sustainable. What I mean by that is that it's easy to be very enthusiastic early on and take up more stuff than one can actually handle. This can often lead to burnout in later stages. It's much more sustainable to work on things iteratively."
+
+## [Kunal Kushwaha](https://github.com/kunal-kushwaha)
+
+Kunal Kushwaha is a core member of the Kubernetes marketing council. He is also a CNCF ambassador and one of the founders of the "CNCF Students" program. He also served as a Communications role shadow during the 1.22 release cycle.
```

Comment:
I'm not sure who the "Kubernetes marketing council" is?  Is this supposed to be Contributor Comms, or some other team?

Reply:
Yeah this is the contributor comms team, it was recently renamed

Reply:
https://github.com/kubernetes/community/blob/master/communication/marketing-team/role-handbooks/council.md

Reply:
Yeah, this was a change made by @mbbroberg (with all our approvals). I will admit, the marketing team has several names at this point. Maybe more consistency throughout the multiple envs would help.